src/planscape/planning/urls.py

```diff
@@ -144,6 +136,8 @@
     ),
     path("user_prefs/", UserPreferencesView.as_view(), name="get_userprefs"),
     path("user_prefs/", UserPreferencesView.as_view(), name="patch_userprefs"),
-    # v2 URLS
+    # DEPRECATE WHEN FRONT-END MIGRATES
```

Comment:
wait im confused, are the v2 the ones we need to deprecate? though this are the new ones